experiments/src/browsergym/experiments/loop.py

```diff
@@ -304,7 +306,7 @@ class StepInfo:
     profiling: StepTimestamps = field(default_factory=StepTimestamps)
     task_info: dict = None
 
-    def from_step(self, env: gym.Env, action: str):
+    def from_step(self, env: gym.Env, action: str, obs_preprocessor: callable):
```

Comment:
maybe add default for obs_preprocessor=None so as not to break older code

Reply:
no need, the method is only called here and I believe it's better here to crash if not provided than potentially creating a silent bug.